src/routes/auth.routes.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-console */
+/* eslint-disable no-undef */
+import express from 'express';
+import { authController } from '../controllers/auth.controller.js';
+import { catchError } from '../untils/catchError.js';
+import { authMiddleware } from '../middlewares/authMiddleware.js';
+
+export const authRouter = new express.Router();
+
+authRouter.post('/registration', catchError(authController.register));
```

Comment:
Ensure that the registration process includes necessary validations and error handling to prevent issues such as duplicate registrations or invalid data.